jest.config.js

```diff
@@ -6,7 +6,8 @@ module.exports = {
   coverageDirectory: "coverage",
   collectCoverageFrom: [
     "src/**/*.ts",
-    "!src/proto/**"
+    "!src/proto/**",
```

Comment:
I think you can delete this, this is a leftover from when we included protobuf directly in the source dir